### ISSUE TITLE
"vagrant up" fails if "apt-get update" is called

### ISSUE THREAD

**Issue by philbates35** [laravel/homestead#1055]
## Versions
  - Vagrant: 2.1.2
  - Provider: Virtualbox 5.2.24
  - Homestead: Homestead git repository v8.0.2, box version 7.0.0.

## Host operating system
Fedora 29

## Homestead.yaml
```yaml
---
ip: "192.168.10.10"
memory: 2048
cpus: 1
provider: virtualbox

authorize: ~/.ssh/id_rsa.pub

keys:
    - ~/.ssh/id_rsa

folders:
    - map: ~/Code
      to: /home/vagrant/Code
      type: "nfs"

sites:
    - map: natours.test
      to: /home/vagrant/Code/natours

databases:
    - homestead

# ports:
#     - send: 50000
#       to: 5000
#     - send: 7777
#       to: 777
#       protocol: udp

# blackfire:
#     - id: foo
#       token: bar
#       client-id: foo
#       client-token: bar

# zray:
#  If you've already freely registered Z-Ray, you can place the token here.
#     - email: foo@bar.com
#       token: foo
#  Don't forget to ensure that you have 'zray: "true"' for your site.
```

## Vagrant destroy & up output
https://gist.github.com/philbates35/7d843776f81520b0ad3984b575fdd803

## Expected behavior
`apt-get update` (triggered by the `vagrant-bindfs` plugin) should have completed successfully, bindfs installed, and vagrant up completed successfully.

## Actual behavior
The bind fs plugin results in a call to `apt-get update` during `vagrant up`, but the `apt-get update` fails with the error:
```
E: Repository 'http://ppa.launchpad.net/nginx/development/ubuntu bionic InRelease' changed its 'Label' value from 'NGINX Mainline' to 'NGINX Mainline (1.15.x)'
```
before it gets to installing `bindfs`.

So although the error happens as a result of the `vagrant-bindfs` plugin being installed, it is the general `apt-get update` command that is failing, which makes believe this is a Homestead isssue, not a bindfs issue.

## Steps to reproduce
1. `vagrant plugin install vagrant-bindfs`
2. `vagrant destroy`
3. `vagrant up`

## References
Are there any other [GitHub issues](https://github.com/laravel/homestead/issues?utf8=%E2%9C%93&q=is%3Aissue) (open or closed) that should be linked here?
I searched and couldn't find anything.

**Comment by svpernova09** [comment]
Does it complete without errors if you remove the bindfs plugin?

**Comment by svpernova09** [comment]
Run this command from inside the VM: `sudo apt-get --allow-releaseinfo-change update`

Then exit and run `vagrant --provision`

**Comment by philbates35** [comment]
Seems like that fixed it. Relevant output here:
```
$ vagrant up
$ vagrant ssh
vagrant@homestead:~$ sudo apt-get update // Just to prove it fails
Get:1 http://ppa.launchpad.net/nginx/development/ubuntu bionic InRelease [15.4 kB]
Get:2 http://security.ubuntu.com/ubuntu bionic-security InRelease [88.7 kB]                                                                                                                                       
Get:3 http://ppa.launchpad.net/ondrej/php/ubuntu bionic InRelease [20.8 kB]                                                                                                                                       
Get:4 https://deb.nodesource.com/node_8.x bionic InRelease [4,622 B]                                                                                                                                              
Hit:5 http://archive.ubuntu.com/ubuntu bionic InRelease                                                                                                                                                           
Get:6 https://packages.microsoft.com/ubuntu/18.04/prod bionic InRelease [3,226 B]                                                                                                     
Get:8 http://archive.ubuntu.com/ubuntu bionic-updates InRelease [88.7 kB]                                                                                           
Hit:7 https://cli-assets.heroku.com/apt ./ InRelease                                                      
Get:9 http://ppa.launchpad.net/ondrej/php/ubuntu bionic/main i386 Packages [43.9 kB]                                                  
Get:10 http://archive.ubuntu.com/ubuntu bionic-backports InRelease [74.6 kB]                             
Get:11 http://ppa.launchpad.net/ondrej/php/ubuntu bionic/main amd64 Packages [43.9 kB]                                     
Get:12 https://packages.microsoft.com/ubuntu/18.04/prod bionic/main amd64 Packages [35.7 kB]
Get:13 http://security.ubuntu.com/ubuntu bionic-security/main amd64 Packages [255 kB]
Get:14 http://security.ubuntu.com/ubuntu bionic-security/main i386 Packages [197 kB]             
Get:15 http://security.ubuntu.com/ubuntu bionic-security/main Translation-en [96.8 kB]                                  
Get:16 http://archive.ubuntu.com/ubuntu bionic-updates/main i386 Packages [446 kB]                                                       
Get:17 http://security.ubuntu.com/ubuntu bionic-security/universe i386 Packages [114 kB]                                                            
Get:18 http://security.ubuntu.com/ubuntu bionic-security/universe amd64 Packages [116 kB]                          
Get:19 http://security.ubuntu.com/ubuntu bionic-security/universe Translation-en [66.2 kB]                        
Get:20 http://security.ubuntu.com/ubuntu bionic-security/multiverse amd64 Packages [3,332 B]                            
Get:21 http://security.ubuntu.com/ubuntu bionic-security/multiverse i386 Packages [3,500 B]                             
Get:22 http://security.ubuntu.com/ubuntu bionic-security/multiverse Translation-en [1,848 B]                       
Get:23 http://packages.blackfire.io/debian any InRelease [23.2 kB]                                                       
Get:24 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 Packages [512 kB]
Get:25 http://archive.ubuntu.com/ubuntu bionic-updates/main Translation-en [193 kB]          
Get:26 http://archive.ubuntu.com/ubuntu bionic-updates/universe amd64 Packages [724 kB]
Get:27 http://archive.ubuntu.com/ubuntu bionic-updates/universe i386 Packages [716 kB]
Get:28 http://archive.ubuntu.com/ubuntu bionic-updates/universe Translation-en [183 kB]
Get:29 http://archive.ubuntu.com/ubuntu bionic-updates/multiverse i386 Packages [6,540 B]
Get:30 http://packages.blackfire.io/debian any/main i386 Packages [1,194 B]        
Get:31 http://archive.ubuntu.com/ubuntu bionic-updates/multiverse amd64 Packages [6,388 B]
Get:32 http://archive.ubuntu.com/ubuntu bionic-updates/multiverse Translation-en [3,452 B]
Get:33 http://archive.ubuntu.com/ubuntu bionic-backports/universe i386 Packages [3,472 B]
Get:34 http://archive.ubuntu.com/ubuntu bionic-backports/universe amd64 Packages [3,472 B]
Get:35 http://packages.blackfire.io/debian any/main amd64 Packages [1,208 B] 
Reading package lists... Done        
E: Repository 'http://ppa.launchpad.net/nginx/development/ubuntu bionic InRelease' changed its 'Label' value from 'NGINX Mainline' to 'NGINX Mainline (1.15.x)'
N: This must be accepted explicitly before updates for this repository can be applied. See apt-secure(8) manpage for details.
vagrant@homestead:~$ sudo apt-get --allow-releaseinfo-change update
Hit:2 http://ppa.launchpad.net/ondrej/php/ubuntu bionic InRelease                                                                                                                                                 
Hit:3 http://archive.ubuntu.com/ubuntu bionic InRelease                                                                                                                                                           
Hit:4 http://security.ubuntu.com/ubuntu bionic-security InRelease                                                                                                                                                 
Hit:5 http://archive.ubuntu.com/ubuntu bionic-updates InRelease                                                                                                                                                   
Hit:6 http://archive.ubuntu.com/ubuntu bionic-backports InRelease                                                                                                                                                 
Get:7 https://deb.nodesource.com/node_8.x bionic InRelease [4,622 B]                                                                                                                                              
Hit:9 https://packages.microsoft.com/ubuntu/18.04/prod bionic InRelease                                                                                                      
Hit:8 https://cli-assets.heroku.com/apt ./ InRelease                                                                                            
Get:1 http://ppa.launchpad.net/nginx/development/ubuntu bionic InRelease [15.4 kB]                  
Get:10 http://ppa.launchpad.net/nginx/development/ubuntu bionic/main amd64 Packages [4,264 B]
Get:11 http://ppa.launchpad.net/nginx/development/ubuntu bionic/main i386 Packages [4,260 B]     
Hit:12 http://packages.blackfire.io/debian any InRelease                 
Fetched 28.5 kB in 2s (16.3 kB/s)
Reading package lists... Done
N: Repository 'http://ppa.launchpad.net/nginx/development/ubuntu bionic InRelease' changed its 'Label' value from 'NGINX Mainline' to 'NGINX Mainline (1.15.x)'
vagrant@homestead:~$ exit
$ vagrant provision
$ vagrant ssh
vagrant@homestead:~$ sudo apt-get update // Now completes with no errors
Hit:1 http://ppa.launchpad.net/nginx/development/ubuntu bionic InRelease                                                                                                                                          
Hit:2 http://security.ubuntu.com/ubuntu bionic-security InRelease                                                                                                                                                 
Hit:3 http://ppa.launchpad.net/ondrej/php/ubuntu bionic InRelease                                                                                                                                                 
Hit:4 http://archive.ubuntu.com/ubuntu bionic InRelease                                                                                                                                                           
Hit:5 http://archive.ubuntu.com/ubuntu bionic-updates InRelease                                                                                                                                                   
Get:6 https://deb.nodesource.com/node_8.x bionic InRelease [4,622 B]                                                                                                                                              
Hit:8 http://archive.ubuntu.com/ubuntu bionic-backports InRelease                                                                                                                
Hit:7 https://cli-assets.heroku.com/apt ./ InRelease                                                                          
Hit:9 https://packages.microsoft.com/ubuntu/18.04/prod bionic InRelease                                                       
Hit:10 http://packages.blackfire.io/debian any InRelease                 
Fetched 4,622 B in 2s (1,948 B/s)
Reading package lists... Done
```
Full output here: https://gist.github.com/philbates35/cf615bd9e09a0b0c16b68ac87be2bdbf

**Comment by svpernova09** [comment]
Good, we had the same issue on another repository not long ago.

**Comment by philbates35** [comment]
For reference, this issue seems to be fixed in version 7.1.0 of the homestead box (laravel/settler).

**Comment by svpernova09** [comment]
@philbates35 Yep! thanks for reporting back :D
